packages/client/src/__tests__/customerDataTriggers.test.ts

```diff
@@ -94,20 +94,25 @@ describe("Customer triggers", () => {
         condition: (data) => Boolean(data),
       });
       // add update to db
-      const updatedCategory = Category.CourseAdults;
+      const updatedCategory = [Category.CourseAdults];
       const updatedSaul = {
         ...saul,
         secretKey,
         category: updatedCategory,
         deleted: true,
       };
       await setDoc(saulDocRef, updatedSaul);
+
       // check `bookings` entry updates
       await waitForCondition({
         documentPath: getBookingsDocPath(organization, secretKey),
         condition: (data) =>
           Boolean(
-            data && data.category === updatedCategory && data.deleted === true
+            data &&
+              data.category.every((cat: Category) =>
+                updatedCategory.includes(cat)
+              ) &&
```

Comment:
Here, if you want to test that the updated bookings doc includes all `updatedCategories`, you wouldn't loop over the bookings doc's categories and check that they're included in `updatedCategories`, but rather the other way around: you want to assert that each of the `updatedCategories` found its way to `data.category`.